src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/sideeffects/HandEventHandler.java

```diff
@@ -0,0 +1,67 @@
+package ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.sideeffects;
+
+import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.entity.Hand;
+import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.entity.Match;
+import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.entity.MatchState;
+import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.repository.HandRepository;
+import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.repository.MatchRepository;
+import java.util.Collection;
+import javax.persistence.EntityNotFoundException;
+import org.springframework.data.rest.core.annotation.HandleAfterSave;
+import org.springframework.data.rest.core.annotation.RepositoryEventHandler;
+import org.springframework.stereotype.Component;
+
+@Component
+@RepositoryEventHandler
+public class HandEventHandler {
+
+  private final MatchRepository matchRepository;
+  private final HandRepository handRepository;
+
+  public HandEventHandler(MatchRepository matchRepository, HandRepository handRepository) {
+    this.matchRepository = matchRepository;
+    this.handRepository = handRepository;
+  }
+
+  @SuppressWarnings("unused")
+  @HandleAfterSave
+  public void onAfterSave(Hand hand) {
+    long idMatch = hand.getMatch().getId();
```

Comment:
What happens if the hand is matched again?
will the match be moved back to PLAYING again?

Reply:
Die hand wird nur einmal gespielt.